testing/ostest/spinlock.c

```diff
@@ -46,10 +46,14 @@ static int g_result = 0;
 
 static FAR void thread_spinlock(FAR void *parameter)
 {
+  int pid = *(int *)parameter;
+
   for (int i = 0; i < 10; i++)
     {
       spin_lock(&lock);
+      printf("pid %d get lock g_result:%d\n", pid, g_result);
```

Comment:
it wrong to call printf inside a spin lock

Reply:
I got it and I add a new rw_spinlock test for future.